v1/pom.xml

```diff
@@ -431,16 +432,16 @@
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value-annotations</artifactId>
-      <version>${autovalue.annotations.version}</version>
+      <version>${autovalue.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.codehaus.jackson</groupId>
-      <artifactId>jackson-core-asl</artifactId>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
```

Comment:
This dependency moved and all the versions on the old naming have the vulnerability, so moving to the new repository name.